### PR TITLE
Update endpoints, fields and resources to API v6

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Updating API fields, endpoints, and resources to v6.
Referring to https://github.com/balena-io/balena-sdk/blob/master/typings/balena-sdk/models.d.ts for updated fields

Change-type: minor
Also, solves #1496 
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
@dfunckt Let me know if there is any more examples I should be adding

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

File: config/dictionaries/resource.json [hunk 1]
```diff
@@ -661,42 +683,111 @@
...
+        "summary": "Get organization by ID",
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/organization(<ID>)",
+        "filters": ""
+      },
+      {
+        "id": "org-by-handle",
+        "summary": "Get organization by handle",
+        "description": "Handles are a simpler way to identify organizations. For example: An organization called Foo Bar Enterprises will have the handle foo_bar_enterprises. For gh_john_doe's Organization the handle will be gh_john_doe.",
```
Comment (dfunckt) [inline config/dictionaries/resource.json:724]:
```suggestion
        "description": "Uniquely identifies an organization. Handles are currently automatically generated and read-only.",
```

File: config/dictionaries/resource.json [hunk 2]
```diff
@@ -801,16 +882,20 @@
...
     "fields": [
       "id",
-      "created_at",
-      "installs__service"
+    "created_at",
+    "device",
+    "should_be_running",
+    "installs__service",
+    "application",
+    "device_service_environment_variable"
```
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply config/dictionaries/resource.json]: Adding the fields the v6 endpoint provides.

File: config/dictionaries/resource.json [hunk 3]
```diff
@@ -4,16 +4,27 @@
...
       "app_name",
       "slug",
-      "commit",
+	  "commit",
+	  "should_be_running__release",
       "application_type",
-      "device_type",
-      "should_track_latest_release",
+	  "should_track_latest_release",
+	  "user__is_member_of__application",
```
Comment (dfunckt) [inline config/dictionaries/resource.json:26]: There's some white-space issue here.

File: config/dictionaries/resource.json (hunk 1 above)
Comment (dfunckt) [reply config/dictionaries/resource.json:724]: This was marked as resolved I suspect accidentally.

File: config/dictionaries/resource.json [hunk 4]
```diff
@@ -661,42 +683,111 @@
...
+    "start_timestamp",
+    "update_timestamp",
+    "end_timestamp",
+    "is_created_by__user",
+    "belongs_to__application",
+    "contains__image",
+    "should_be_running_on__application",
+    "is_running_on__device",
+    "should_be_running_on__device",
+    "release_tag"
```
Comment (dfunckt) [inline config/dictionaries/resource.json:774]: White-space issues here as well.

**Review by dfunckt — COMMENTED** [review]
Looks good @vipulgupta2048 👍

File: config/dictionaries/resource.json (hunk 1 above)
Comment (vipulgupta2048) [reply config/dictionaries/resource.json:724]: My bad, didn't push the commit fixing this bit. Coming right up.

File: config/dictionaries/resource.json (hunk 3 above)
Comment (vipulgupta2048) [reply config/dictionaries/resource.json:26]: Converted all to spaces, will check this bit after pushing to GitHub.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**